Quiesce test failing for pre-squid

Possibly due to the new feature being backported to squid branch:

```
 === RUN   TestFSQuiesce
    fs_quiesce_reef_test.go:22: 
        	Error Trace:	/go/src/github.com/ceph/go-ceph/cephfs/admin/fs_quiesce_reef_test.go:22
        	Error:      	Expected nil, but got: &admin.FSQuiesceInfo{Epoch:5, SetVersion:1, Sets:map[string]admin.QuiesceInfo{"ac47ac80":admin.QuiesceInfo{Version:1, AgeRef:0, State:admin.QuiesceState{Name:"TIMEDOUT", Age:0}, Timeout:0, Expiration:0, Members:map[string]admin.QuiesceInfoMember{"file:/volumes/_nogroup/quiesceMe/bf7b4f2b-e605-40f0-8103-4b7f692440ad":admin.QuiesceInfoMember{Excluded:false, State:admin.QuiesceState{Name:"QUIESCING", Age:0}}}}}}
        	Test:       	TestFSQuiesce
    fs_quiesce_reef_test.go:24: 
        	Error Trace:	/go/src/github.com/ceph/go-ceph/cephfs/admin/fs_quiesce_reef_test.go:24
        	Error:      	Should be true
        	Test:       	TestFSQuiesce
--- FAIL: TestFSQuiesce (0.09s)
=== RUN   TestServerSentinel
```

https://github.com/ceph/go-ceph/actions/runs/8398382664/job/23003150402

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -70,7 +70,7 @@ jobs:
         - "pacific"
         - "quincy"
         - "reef"
-      # - "squid"
+        - "squid"
         - "pre-quincy"
         - "pre-reef"
         - "pre-squid"

